minkindr_conversions/include/minkindr_conversions/kindr_msg.h

```diff
@@ -76,7 +76,8 @@ void quaternionMsgToKindr2D(
   kindr::minimal::RotationQuaternionTemplate<Scalar> quat;
   quaternionMsgToEigen(msg, quat.toImplementation());
   const kindr::minimal::AngleAxisTemplate<Scalar> angle_axis(quat);
-  kindr->angle() = angle_axis.angle();
+  const Scalar sign = angle_axis.axis()(2) >= Scalar(0) ? Scalar(1) : Scalar(-1);
+  kindr->angle() = angle_axis.angle() * sign;
```

Comment:
This should be equivalent (since `angle_axis.angle()` is non-negative) and cleaner:
```suggestion
  kindr->angle() = std::copysign(angle_axis.angle(), angle_axis.axis()(2));
```

Reply:
Nice, that is much cleaner.